share/functions/__fish_print_VBox_vms.fish

```diff
@@ -0,0 +1,28 @@
+function __fish_print_VBox_vms
+    set -l print_names true
+    set -l print_uuids true
+
+    if set -q argv[1]
+        if test $argv[1] = 'nouuids'
+            set print_uuids false
+        end
+        if test $argv[1] = 'nonames'
+            set print_names false
+        end
+    end
+
+    set -l lines (VBoxManage list vms | string match -r '"(.*)" {(.*)}')
```

Comment:
Personally, I'd like some example output as a comment here (though that's possibly a bit hypocritical of me).